tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/expectedtype/TableConstructorExpressionTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Shouldn't the copyright header be updated?